src/analysis/parsing/expression.rs

```diff
@@ -92,6 +92,7 @@ impl TreeElement for BinaryExpressionContent {
     fn subs(&self) -> TreeElements<'_> {
         create_subs!(&self.left, &self.operation, &self.right)
     }
+    // fn style_check(&self, acc: &mut Vec<DMLStyleError>, rules: &CurrentRules, mut aux: AuxParams) {}
 }
```

Comment:
@alecalvop uncommenting this empty style_check function causes IN5/paren_expr test to fail